libbeat/docs/shared-docker.asciidoc

```diff
@@ -167,6 +167,7 @@ docker run -d \
   --volume="$(pwd)/{beatname_lc}.docker.yml:/usr/share/{beatname_lc}/{beatname_lc}.yml:ro" \
   --volume="/var/lib/docker/containers:/var/lib/docker/containers:ro" \
   --volume="/var/run/docker.sock:/var/run/docker.sock:ro" \
+  --volume="registry:/usr/share/{beatname_lc}/data:rw \  
```

Comment:
looks like the closing quote is missing

Reply:
Thanks for catching that! Here's a PR to add the missing quote: https://github.com/elastic/beats/pull/37051